src/mono/mono/metadata/domain-internals.h

```diff
@@ -665,6 +665,11 @@ void mono_assembly_cleanup_domain_bindings (guint32 domain_id);
 
 MonoJitInfo* mono_jit_info_table_find_internal (MonoDomain *domain, gpointer addr, gboolean try_aot, gboolean allow_trampolines);
 
+typedef void (*MonoJitInfoFunc) (MonoJitInfo *ji, gpointer user_data);
+
+void
+jit_info_table_foreach (MonoDomain *domain, MonoJitInfoFunc func, gpointer user_data);
```

Comment:
nit: `mono_jit_info_table_foreach`

Reply:
OK, will probably add _internal to it as its used for naming of similar methods.